### ISSUE TITLE
Conformer/ ContextNet RNN-Transducer loss computation

### ISSUE THREAD

**Issue by pyyush** [tensorflow/lingvo#236]
Does lingvo include the RNN-Transducer loss computation ?

**Comment by protoget** [comment]
No the RNNT loss impl isn't included.

**Comment by galv** [comment]
I can probably implement RNN-T loss function, at some point, if there is enough interest. It is straight forward to implement if you have an understanding of the CTC loss function's implementation in tensorflow. Can people respond with upvotes on this comment if you are seriously interested in it? Note that there would be absolutely no guarantees that is performant, or that I would even make the time to do it.

I'm not a developer of lingvo to clarify, but this appears to be a common ask from a lot of people.

**Comment by protoget** [comment]
Re @galv on our side we don't have a plan to OSS the impl. So 'plz go ahead' from my side.

**Comment by Mddct** [comment]
Any plan for this？
